lua/nvim-treesitter/configs.lua

```diff
@@ -528,6 +534,42 @@ function M.init()
   end
 end
 
+-- If parser_install_dir is not nil is used or created.
+-- If parser_install_dir is nil try the package dir of the nvim-treesitter
+-- plugin first, followed by the "site" dir from "runtimepath". "site" dir will
+-- be created if it doesn't exist. Using only the package dir won't work when
+-- the plugin is installed with Nix, since the "/nix/store" is read-only.
```

Comment:
unrelated to this PR: did somebody plan to create a Nix package for the parsers. We could provide the install info as JSON and installing via a nix flake would be the most natural thing to do for Nix users. Especially when they can profit from binary caching.

Reply:
I think Nix should take care of Nix users. I am _very_ opposed to catering to a single distribution that insists to be different (for whatever good reason they have, but that is on them, not us).

Reply:
But this is what I was saying. That Nix users should install it the nix way their own nix flake. I was asking whether there are any known efforts.